### PR TITLE
Log cause message of causes without stack traces

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: akka-actor/src/main/scala/akka/event/Logging.scala [hunk 1]
```diff
@@ -1020,7 +1020,7 @@ object Logging {
    */
   def stackTraceFor(e: Throwable): String = e match {
     case null | Error.NoCause ⇒ ""
-    case _: NoStackTrace      ⇒ " (" + e.getClass.getName + ")"
+    case _: NoStackTrace      ⇒ s" (${e.getClass.getName}: ${e.getMessage})"
```
Comment (ktoso) [inline akka-actor/src/main/scala/akka/event/Logging.scala]: good idea

**Review by ktoso — APPROVED** [review]
LGTM

**Review by johanandren — APPROVED** [review]
LGTM

**Review by patriknw — COMMENTED** [review]
+1

**Pull request merged**